multi/imxrt-multi/imxrt1170.h

```diff
@@ -308,16 +308,32 @@
 
 static inline int common_setClock(int clock, int div, int mux, int mfd, int mfn, int state)
```

Comment:
Maybe the function should not be inlined ? I don't know why this tendency is to `static inline` everything here, especially since this function could be reused by calling rather than by inlining.